lib/Backends/CPU/CPUBackend.cpp

```diff
@@ -168,13 +172,13 @@ bool CPUBackend::isOpSupported(Kinded::Kind opKind, ElemKind elementTy) const {
     case Kinded::Kind::TileNodeKind:
     case Kinded::Kind::TopKNodeKind:
     case Kinded::Kind::TransposeNodeKind:
-      return true;
+      return 1;
```

Comment:
can you make this a named const (MinimalCostVal, there might be a better name)?

Reply:
Yes I think this would help with clarity to have a name for this value. Not sure if minimal is right though because I guess somethings may be 0 if they are "free" like the Relu in ConvRelu example?

Reply:
well it depends, on CPU if not fused `relu` is not "free".

Reply:
I added `MinimumUnitCostVal` and replaced all `1`s with it. If we keep it I'm thinking backends would use this minimum unit cost to define other costs, i.e. as some multiple of that. It seems a little silly since it is just `1` and I'm not sure we would ever change it from `1`, but perhaps this preferable to just specifying `1`? Let me know what you think.

I also added `FreeCostVal = 0` to differentiate from the minimum, however it's not used anywhere, so not sure it's worth including for now.

Reply:
> It seems a little silly since it is just 1 and I'm not sure we would ever change it from 1, but perhaps this preferable to just specifying 1?

hopefully name should give slightly more context than plain '1'.

> I also added FreeCostVal = 0 to differentiate from the minimum, however it's not used anywhere, so not sure it's worth including for now.

up to you, we can introduce that once there is a need.

Reply:
>  It seems a little silly since it is just 1 and I'm not sure we would ever change it from 1, but perhaps this preferable to just specifying 1?

@jfix71 I don't think it's too silly, I imagine it would give enough context to make it pretty easy to understand what's being expressed